scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -271,6 +271,12 @@ class PathType(Enum):
     HTTP = "http"    # <UNSUPPORTED> Model files hosted on a HTTP endpoint.
     AZUREBLOB = "azureblob"  # Model files hosted on an AZUREBLOB blobstore with public read access.
 
+class DataType(Enum):
+    """Enum for data types supported for data publishing."""
+
+    URI_FILE = "uri_file"  
+    URI_FOLDER = "uri_folder"      
+    MLTABLE = "mltable"

```

Comment:
Would you please move this enum down just above `DataConfig`? Classes in this file appear close to where they're used.

Reply:
Done. Thanks Louie